jenkins_jobs/trigger_host_os_build_from_builds_repo.groovy

```diff
@@ -23,6 +23,7 @@ job('trigger_host_os_build_from_builds_repo') {
       extensions {
 	commitStatus {
 	  context('Build Host OS')
+	  statusUrl('${JENKINS_URL}/job/build_host_os/${TRIGGERED_BUILD_NUMBER_build_host_os}')
```

Comment:
I have no idea where TRIGGERED_BUILD_NUMBER_build_host_os comes from, you could mention it somewhere